install_files/securedrop-grsec/etc/default/grub.d/50-mds-smt.cfg

```diff
@@ -0,0 +1,3 @@
+# disables smt and provide full mds mitigations
+# see https://www.kernel.org/doc/html/latest/admin-guide/hw-vuln/mds.html
+GRUB_CMDLINE_LINUX="$GRUB_CMDLINE_LINUX mds=full,nosmt"
```

Comment:
The `nosmt` settings may have a modest performance impact, but it appears acceptable: restricting the application processes to the number of cores (i.e. 2 on most recommended setups) for simultaneous execution shouldn't be a significant performance bottleneck, especially considering the other limitations such as network and disk IO. Even though we don't have quantitative metrics on application behavior prior to merge, it's worth flagging for consideration during QA. The typical submit, check, reply, check workflow should work just fine, with no appreciable slowdown. 

The rationale for disabling SMT is that it will help guard against future vulnerabilities in the same class. If the past 18 months have been any indication, we're likely not past the reactive cycle based on disclosures, so taking a more defensive posture, even at the cost of a slight performance hit, is well warranted.